tree/ntuple/v7/test/ntuple_zip.cxx

```diff
@@ -13,7 +13,7 @@ TEST(RNTupleZip, Basics)
 
 TEST(RNTupleZip, Empty)
 {
-   char x;
+   char x = 0;
    char z;
```

Comment:
So it complains for `x` but not for `z`? Interesting...

Reply:
Yes, I think because the first parameter is a `const` pointer, so you are not allowed to write. The only reason to pass in a `const` pointer is to read from it (which we don't do here because the size is 0).